app/controllers/login.js

```diff
@@ -30,11 +30,21 @@ function login(req, res, next) {
 
 			// Return that user if they exist
 			if (user) {
-				auth.signToken({user_id: user.attributes.id}, (err, userJwt) => {
-					return response.ok(res, {
-						jwt: userJwt
-					});
-				});
+				let userPromise;
+				// Update the user's name if it changed
```

Comment:
Move this comment above the `userPromise` declaration.
Also it should mention that the name changed on Orcid, like "Update the user's name if they changed it in Orcid's system`